gdkx11-sys/src/lib.rs

```diff
@@ -0,0 +1,416 @@
+// This file was generated by gir (https://github.com/gtk-rs/gir)
+// from gir-files (https://github.com/gtk-rs/gir-files)
+// DO NOT EDIT
+
+#![allow(non_camel_case_types, non_upper_case_globals, non_snake_case)]
+#![allow(clippy::approx_constant, clippy::type_complexity, clippy::unreadable_literal)]
+
+extern crate libc;
+extern crate gdk_sys as gdk;
+extern crate gdk_pixbuf_sys as gdk_pixbuf;
+extern crate gio_sys as gio;
+extern crate glib_sys as glib;
+extern crate gobject_sys as gobject;
+extern crate pango_sys as pango;
+// manual import
+extern crate x11;
```

Comment:
This is manually added here and we probably want to automate that or make it more obvious so that things don't just break next time someone regenerates